src/screens/UScreenJukebox.pas

```diff
@@ -1673,18 +1679,23 @@ function TScreenJukebox.ParseInput(PressedKey: Cardinal; CharCode: UCS4Char;
             Pause;
         end;
 
-        SDLK_TAB: // change visualization preset
+        SDLK_TAB:
         begin
-          if fShowVisualization then
-            fCurrentVideo.Position := now; // move to a random position
-
-          if (fShowWebcam) then
+          if (SDL_ModState = KMOD_LCTRL) then // change visualization preset
```

Comment:
please adjust the readme.md accordingly

Reply:
See d11d8f55ed9f80b3f3dd06a5af9af016fdea7461